api/swagger.yml

```diff
@@ -59,17 +59,21 @@ externalDocs:
   description: Find out more about Hermez network.
   url: 'https://hermez.io'
 servers:
-  - description: Hosted mock up
-    url: https://apimock.hermez.network/v1
-  - description: Localhost mock Up
-    url: http://localhost:4010/v1
+  - description: Hosted mock up, returns fake data usefull for development
+    url: https://apimock.hermez.network
+  - description: Localhost mock up, returns fake data usefull for development
+    url: http://localhost:4010
+  - description: Testnet (Rinkeby) server
+    url: https://api.testnet.hermez.io
+  - description: Mainnet (Ethereum) server, use it carefully, specially if attempting to send transactions. You could lose money!
+    url: https://api.hermez.io
 tags:
   - name: Coordinator
     description: Endpoints used by the nodes running in coordinator mode. They are used to interact with the network.
   - name: Explorer
     description: Endpoints used by the nodes running in explorer mode. They are used to get information of the netwrok.
 paths:
-  '/account-creation-authorization':
+  '/v1/account-creation-authorization':
```

Comment:
Checking what it says at the spec that you referenced (https://swagger.io/docs/specification/api-host-and-base-path/), `v1/` should be part of the `server URL` and not of the `endpoint`:
```
    https://api.example.com/v1/users?role=admin&status=active
    \________________________/\____/ \______________________/
             server URL       endpoint    query parameters
                                path
```

Reply:
Ok, in this case I'd prefere to add a reverse proxy in the `docker-compose` to fix this, as I've already tried many different options and didn't liked any of them.

Also, another option is to completley remove the scripts needed to spin up the documentation and mock server from this repo. Now that the repo is open it should be straight forward